tools/omero/omero_metadata_upload/.shed.yml

```diff
@@ -0,0 +1,16 @@
+categories:
```

Comment:
Can you move  all omero related tools to `tools/omero/` and use the dhed.yaml of the upload tool

Reply:
https://github.com/Helmholtz-UFZ/galaxy-tools/pull/36#discussion_r1699532878